obspy/io/quakeml/core.py

```diff
@@ -561,6 +561,7 @@ def _origin(self, element, arrivals):
         obj.composite_times = self._composite_times(element)
         obj.quality = self._origin_quality(element)
         obj.origin_type = self._xpath2obj('type', element)
+        obj.region = self._xpath2obj('region', element)
```

Comment:
Yea apparently this was missing. We should generate a full QuakeML file like we did for StationXML (https://github.com/obspy/obspy/blob/master/obspy/io/stationxml/tests/data/full_random_stationxml.xml) to avoid this kind of problem in the future.

@barsch I think you generated the stationxml test file. Could you do the same for QuakeML?